src/benchmarks/gc/GC.Infrastructure/Configurations/ASPNetBenchmarks/ASPNetBenchmarks.yaml

```diff
@@ -3,6 +3,7 @@ runs:
     corerun: C:\CoreRuns\Empty\
     environment_variables:
       COMPlus_GCServer: 1
+    framework_version: net6.0
```

Comment:
I'm curious about the choice of 6.0 here. My guess would have been that samples include the latest version.

Reply:
This was just an example to show case that one can override the framework version at the level of the run.